source/playbooks/work-with-tasks.rst

```diff
@@ -8,13 +8,13 @@ In some workflows, there are time constraints on tasks and others may have more
 
 To assign a due date to a task, select the **Toggle Run Details** icon to open the **Run Details** screen. Hover over the task you’d like to edit and select the calendar icon to assign a due date. Due dates can be used to sort tasks in the run overview.
 
-When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks to-do`` slash command.
+When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks todo`` slash command.
 
 Due dates can be entered in text (e.g., "two minutes ago") or numerically (e.g., "15 March 2023").
 
 Task inbox
 ----------
 
-In addition to the daily digest, you also have access to a task inbox. To view it, open the playbooks tab. 
+In addition to the daily digest, you also have access to a task inbox. The task inbox provides you with a cross-run overview of the tasks you're accountable for, sorted by due date. You can access and complete each task directly, without having to visit the individual runs.
```

Comment:
not sure if it's a good idea to make this more complex, but there are more interesting things that the user could do with each task in the context of task inbox:
- change assignee to change accountability -> disappear from task inbox
- change duedate -> change urgency

What I'm trying to explain is, if this resource mix "what I'm accountable for" + "sense of urgency" the main actions are:
- finish/skip the task by myself
- assign to another person 
- change the sense of urgency

Reply:
Great suggestions, thank you!